### PR TITLE
Adding velocity map into GeometryBufferRenderer and added object based motion blur

### PR DESCRIPTION
**Pull request by julien-moreau**

### CODE REVIEW

**Comment by deltakosh** [comment]
Can you fix the conflict?

File: src/PostProcess/babylon.motionBlurPostProcess.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                    return;
+                }
+
+                this._geometryBufferRenderer.enableVelocity = true;
+            }
+
+            this.onApply = (effect: Effect) => {
+                effect.setVector2("screenSize", new Vector2(this.width, this.height));
+
+                this._motionScale = scene.getEngine().getFps() / 60.0;
```
Comment (deltakosh) [inline src/PostProcess/babylon.motionBlurPostProcess.ts]: you should use scene.getAnimationRatio()

Comment (julien-moreau) [reply src/PostProcess/babylon.motionBlurPostProcess.ts]: Good catch!

**Comment by Hersir88** [comment]
Would be nice to add demo for this, but could be added in other MR

File: dist/preview release/what's new.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 - Added support for [parallel shader compilation](https://www.khronos.org/registry/webgl/extensions/KHR_parallel_shader_compile/) ([Deltakosh](https://github.com/deltakosh))
 - Added FlyCamera for free navigation in 3D space, with a limited set of settings ([Phuein](https://github.com/phuein))
+- Added Object Based Motion Blur post-process ([julien-moreau](https://github.com/julien-moreau))
```
Comment (deltakosh) [inline dist/preview release/what's new.md]: Please update it with the doc when done ;)

Comment (julien-moreau) [reply dist/preview release/what's new.md]: Will put the link when done then :)

**Comment by julien-moreau** [comment]
@Hersir88 a demo is available here for instance: https://www.babylonjs-playground.com/#0CZQUU#1
Will add the documentation tomorrow! :)

**Pull request merged**